client/src/main/java/org/eclipse/hono/client/impl/AbstractRequestResponseClient.java

```diff
@@ -527,6 +566,8 @@ private Message createMessage(final String subject, final Map<String, Object> ap
     /**
      * Creates a request message for a payload and sends it to the peer.
      * <p>
+     * This method uses the default target address of this client as message address.
```

Comment:
this method uses the *getMessageTargetAddress* method to determine the value of the message's *to* property ...